core/scripts/validate_domain.py

```diff
@@ -68,17 +78,27 @@ def validate_button_payloads(english_responses: dict, french_responses: dict) ->
         english_response = english_responses[response_id]
         french_response = french_responses[response_id]
 
-        if get_buttons(english_response) != get_buttons(french_response):
-            button_mismatches.add(response_id)
+        try:
+            if get_buttons(english_response) != get_buttons(french_response):
+                button_mismatches.add(response_id)
+        except ResponseVariantsNotSupported:
+            has_response_variants.add(response_id)
+            continue
 
+    if has_response_variants:
+        log.warn(
+            "Your domain has response variants which are not supported by this tool.",
+            response_ids=sorted(has_response_variants),
+        )
```

Comment:
We should probably check if these response variants are in `utter_qa_sample_` responses, and then raise, instead of displaying a warning. A warning we know will always be there with a long list won't catch our attention and won't keep us from making mistakes

Reply:
Done